client/src/app/app.component.html

```diff
@@ -1,11 +1,19 @@
-<main id="app__main" class="container-fluid h-100">
-  <header class="test">
-<p>Welcome to app!</p>
-    </header>
-  <body class="app__main h-75">
+
+<main id="app__main" class="h-100">
+  <header class="header">
+        <nav class="nav nav-pills flex-column flex-sm-row">
+            <a class="flex-sm-fill text-sm-center nav-link active" href="#">Active</a>
+            <a class="flex-sm-fill text-sm-center nav-link" href="#">Link</a>
+            <a class="flex-sm-fill text-sm-center nav-link" href="#">Link</a>
+            <a class="flex-sm-fill text-sm-center nav-link disabled" href="#">Disabled</a>
+          </nav>
+      </header>
+  <div class="app__main container-fluid h-75">
```

Comment:
The [Bootstrap class](https://v4-alpha.getbootstrap.com/utilities/sizing/) `h-75` here applies a rule stating the element to have a height of 75% of that of the parent container, thus splitting the header and footer sibling tags into a height of 12.5% apiece. In the design document for this effort, the requirements specify that the header and footer get a height of 10% apiece. What gives?